pylint/extensions/consider_refactoring_into_while_condition.py

```diff
@@ -0,0 +1,93 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""Looks for try/except statements with too much code in the try clause."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from astroid import nodes
+
+from pylint import checkers
+from pylint.checkers import utils
+from pylint.interfaces import HIGH
+
+if TYPE_CHECKING:
+    from pylint.lint import PyLinter
+
+
+class ConsiderRefactorIntoWhileConditionChecker(checkers.BaseChecker):
+    """Checks for instances where while loops are implemented with a constant condition
+    which.
+
+    always evaluates to true and the first statement(s) is/are if statements which, when
```

Comment:
```suggestion
    always evaluates to truethey and the first statement(s) is/are if statements which, when
```

Reply:
truthy

Reply:
done

Reply:
it's causing an error in CI checks: https://github.com/PyCQA/pylint/actions/runs/4026257164/jobs/6920556064#step:9:50

should i rollback?

Reply:
We should add truthy in the pyenchant dict :)

Reply:
done, sorry i didn't know about pyenchant dict